### PR TITLE
fix(tooltip): hide tooltip when touched outside of a component

### LINKED ISSUE #276: Tooltips are stuck on touchscreen
> **Please do not disclose security vulnerabilities as issues. See our [security policy](../../SECURITY.md) for responsible disclosures.**

### Describe the bug

Tooltips are stuck on touchscreen

### To reproduce

1. Set up 'Click on a tooltip with small screen'
2. Change view or click on something else
3. See the Tooltips are stuck on touchscreen

### Screenshots

image

### PR DESCRIPTION
**Pull request by HitomiWin**
Fixes: #276

### Describe your changes

Imported useClickOutside from 'react-hooks-shareable' and added it to onPointerDown event.

### Issue ticket number and link

- Fixes #276 

### Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have verified that the code builds perfectly fine on my local system
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have verified that my code follows the style already available in the repository
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by danielkaxis** [comment]
We break current behaviour with this implementation. On mobile devices when the user scrolls, the user could scroll for a small distance before the tooltip disappeared. The motivation for this was that when you are using a mobile device you touch the phone all the time and the tooltip would just disappear. We should discuss this further on how to do.

**Comment by HitomiWin** [comment]
> We break current behaviour with this implementation. On mobile devices when the user scrolls, the user could scroll for a small distance before the tooltip disappeared. The motivation for this was that when you are using a mobile device you touch the phone all the time and the tooltip would just disappear. We should discuss this further on how to do.

See the comment [FTWEB-5607](https://jira.se.axis.com/browse/FTWEB-5607?focusedCommentId=2046793&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2046793)

I think it's ok to disappear soon the tooltip according to the definition of tooltip.

File: packages/ui-tests/src/coreComponents/Tooltip.spec.ts [hunk 1]
```diff
@@ -85,25 +85,15 @@ context('Tooltip mobile device', () => {
...
-      .trigger('touchstart', {
-        touches: [{ clientX: 0, clientY: 0, identifier: 0 }],
-      })
-
-    cy.get(`[data-cy=${data.textDataCy}]`)
-      .parent()
-      .trigger('touchmove', {
-        changedTouches: [{ clientX: 151, clientY: 151, identifier: 0 }],
-      })
+    // Touch the swcreen again
```
Comment (danielkaxis) [inline packages/ui-tests/src/coreComponents/Tooltip.spec.ts:95]: :)

**Review by danielkaxis — CHANGES_REQUESTED** [review]

File: packages/ui-tests/src/coreComponents/Tooltip.spec.ts (hunk 1 above)
Comment (HitomiWin) [reply packages/ui-tests/src/coreComponents/Tooltip.spec.ts]: sorry. fixed it.

**Review by danielkaxis — APPROVED** [review]

**Pull request merged**